WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2054,3 +2054,17 @@ ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
 MetaPropertyTable       =
 MetaPropertyTableCell   = padding:@SkinWrapperPadding@ @WidgetInternalPadding@; padding-top:0;
 
+#######################################################################
+#   Task List view
+#######################################################################
+TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right: 12px;
+TaskListViewNewTaskBannerWrapper                            = @Text-hint@ @InactiveCursor@ @FontSize-big@ padding:16px 0 16px 0; margin-left:6px;
+TaskListViewHeaderItemLabel                                 = @FontSize-slightly-big@ @Text-light@ padding: 10px 0px 10px 11px;
+TaskReadingPaneRight-ListViewItemCheckboxCell               = padding-left:10px; vertical-align:middle;
+TaskReadingPaneRight-ListViewItem                           = height:80px;
+TaskReadingPaneRight-ListViewItemSubject                    = padding-top:32px;
```

Comment:
can we try having vertical-align:middle or display: table cell. Using particular value doesn't look like very good to me.